tests/conftest.py

```diff
@@ -48,16 +48,16 @@ def n_subsystems(request):
     return request.param
 
 
-@pytest.fixture(scope="function")
-def qubit_device_1_wire():
-    return LightningQubit(wires=1)
+@pytest.fixture(scope="function", params=[np.complex64, np.complex128])
+def qubit_device_1_wire(request):
+    return LightningQubit(wires=1, c_dtype=request.param)
```

Comment:
Instead of inserting datatype directly to a statevector, we now set the device datatypes. Test suits are updated accordingly.